packages/twenty-front/src/modules/object-record/record-merge/utils/recordMergeUtils.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  IconSquareNumber1,
+  IconSquareNumber2,
+  IconSquareNumber3,
+  IconSquareNumber4,
+  IconSquareNumber5,
+  IconSquareNumber6,
+  IconSquareNumber7,
+  IconSquareNumber8,
+  IconSquareNumber9,
+} from 'twenty-ui/display';
+
+export const getPositionWordLabel = (index: number): string => {
+  const labels = [
```

Comment:
Can you try to use typing to make sure it's not missing any label, maybe something like `FixedLengthArray<string, typeof OUR_VAR_WITH_NUMBER_OF_ITEMS_HERE>`